### DISCUSSION TITLE
skoala/ms-gateway/api/add-api/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1583] [Announcements]
# skoala/ms-gateway/api/add-api/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/skoala/ms-gateway/api/add-api/

**Comment by liyuerich** [comment]
API 分组：选择 API 所属的分组组名。如果输入不存在的分组名称，则自动创建一个新的分组。

有个default 分组，从哪里来的呢？

**Reply by Michelle951** [reply]
default是自动创建的默认分组，类似于新集群下面无论如何都有一个 default 命名空间一样
